sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -1645,7 +1644,7 @@ export class ServiceBusAdministrationClient extends ServiceClient {
    * @throws `RestError` with code and statusCode representing the standard set of REST API errors.
    */
   async updateSubscription(
-    subscription: SubscriptionProperties,
+    subscription: WithResponse<SubscriptionProperties>,
```

Comment:
What is the raw response that you need here? Shouldn't it just take a SubscriptionProperties?

Reply:
Like are we trying to say:

updateSubscription(subscription: SubscriptionProperties | WithResponse<SubscritionProperties>)?

Reply:
`SubscriptionProperties` would be enough, but I had two reasons for `WithResponse<SubscriptionProperties>`
- Users don't get just the `SubscriptionProperties` from any of the methods, all the returned values have `_response`. So, it makes sense to ask for `WithResponse<SubscriptionProperties>` to remove any confusion.
- eTag support - If we ever add eTag support, instead of adding eTag property to the `{Entity}Properties` to use in the update methods, we could simply make it part of `_response`, and use it in the update methods. For this reason, `WithResponse<Props>` made more sense.

Reply:
I think this is fine but I believe, even for etag support, that you'd be able to "ignore" etags (ie, even when they are introduced they wouldn't be _required_?) and just overwrite whatever is there with what you want.

Reply:
We wanted to enforce the eTags once the service starts to support to avoid race conditions or overriding the properties of the queue.